README.md

```diff
@@ -1,216 +1,265 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Mobile Application
+# Compose Multiplatform mobile application
 
-Use this template to start developing your own Compose Multiplatform application targeting Android and iOS (Alpha).
+> Compose Multiplatform is in Alpha. It may change incompatibly and require manual migration in the future.
+
+You can use this template to start developing your own [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) application.
```

Comment:
I would explain what you will get and learn after completing this tutorial. Ans then add that `You can also use this template to start developing your own mobile application targeting iOS and Android`.

Reply:
I think that's backwards, it's a template first, getting started second. We can still mention it, but "Use this to start writing your Compose Multiplatform mobile app" should take precedence

Reply:
I like it!